### ISSUE TITLE
 High-DPI support

### ISSUE THREAD

**Issue by skwerlman** [TanninOne/modorganizer#658]
I'd really like to see support for high-DPI (~288) screens added, since currently the images, column headers, and default column widths are far too small:

![image](https://cloud.githubusercontent.com/assets/6954107/14694885/c40d5de8-073a-11e6-8300-c30b272402aa.png)

**Comment by filmcriticbeta** [comment]
AFIAK, MO is currently built against version 5.5 of qt, and Hi-DPI support requires version 5.6.  On applications built with QT 5.6, end users can add QT_AUTO_SCREEN_SCALE_FACTOR=1 to their windows system variables to force qt to enable Hi DPI support.  However YMMV.

**Comment by skwerlman** [comment]
I don't believe high DPI support is specific to qt 5.6+, seeing as cockatrice, another qt app, has begun implementing high DPI support for both qt 5.2+ and qt 4.

**Comment by TanninOne** [comment]
MO is themeable through the qss files in the stylesheets folder. it's basically css.
With that you can make fonts/buttons larger and so on. Unfortunately I'm myself not an expert on the syntax and don't own a hi-dpi display

**Comment by skwerlman** [comment]
I'll look into modifying the theme I'm currently using. I'm unsure if simply theming it will be sufficient, but then again, I know next to nothing about qt.

If i can get it to work, I'll open a PR.

**Comment by skwerlman** [comment]
Alright, my findings so far:

Neither the default theme, nor the fusion theme can be modified.

Neither DownloadListWidget nor DownloadListWidgetCompact is themeable, though I'm not sure what the difference between them is. Either way, it prevents me from fixing this:
![image](https://cloud.githubusercontent.com/assets/6954107/14764189/1d0ce564-097d-11e6-947f-b455629a0b1f.png)

The scrollbars being too narrow were an easy fix. I just set their width to `1em` instead of `13px`; this _should_ also work on normal (96dpi) screens. Same for the height/width of checkboxes.

I have no idea which class is the filter widget; once I know what it is (and assuming it's themeable), it should be easy to fix.

There are a couple other issues which I don't think can be fixed in the qss:

Most dialogues are too small when created (a couple are are so small qt is unable to create them at the specified size, and so makes them larger). This could be fixed (i think) by setting the dialogue sizes to a % of the screen, rather than a fixed pixel size.

Table header widths should start as a % of the list's width, instead of a fixed pixel size. This is also not changeable through qss (so far as i can tell)

Finally, all of the provided images are too small for high dpi screens.  I am unsure of how to fix this without losing image quality in one case or the other, seeing as many are too complex to be reasonably replicated as `.svg`s, which is what's done in other projects (unless someone's up to redo all the images from scratch).

**Comment by filmcriticbeta** [comment]
Just some feedback, but 2.06 fixes around 90% of the scaling issues ( I'm using 3x scaling on my display ).  

![hpdpi beta2](https://cloud.githubusercontent.com/assets/18420548/15641612/1a0fb5ec-263a-11e6-80a8-59f86aba2179.png)
![hidpi beta2 dracula](https://cloud.githubusercontent.com/assets/18420548/15641815/5757f9b8-263b-11e6-90ac-21d21f4dea27.png)
![beta2 hidpi dark](https://cloud.githubusercontent.com/assets/18420548/15641821/5e202982-263b-11e6-9bcd-771f833fc1b8.png)
![beta2 hidpi standard](https://cloud.githubusercontent.com/assets/18420548/15641825/634883f0-263b-11e6-8725-083dad69cc67.png)

The main remaining issues are:

* Some skins exhibit some minor vertical truncation of text. ( dracula is the most compatible )
* Size of the icons
* Some truncation of long strings
* When opening a food based installer, the dialog window is too small.  An error message is shown in the log window ( see screenshot).   Manually resizing the dialog fixes the issue.

![hidpi beta2 fomod](https://cloud.githubusercontent.com/assets/18420548/15641660/5edfe476-263a-11e6-98cb-49219fff00bc.png)
